src/Core/AppleNative/PlatformInterop/PlatformInterop/CALayer+AutoSizeToSuperLayer.h

```diff
@@ -0,0 +1,8 @@
+#import <UIKit/UIKit.h>
+#import <QuartzCore/QuartzCore.h>
+
+@interface CALayer (AutoSizeToSuperLayer)
```

Comment:
The category should have a MAUI prefix to avoid potential name clashes with third-party libraries:


```suggestion
@interface CALayer (MauiAutoSizeToSuperLayer)
```